src/web/modules/transactions/discrepancies/validateLedger.http.ts

```diff
@@ -10,7 +14,36 @@ export async function validateLedgerTransaction(
   try {
     const ledgerEntry = await Ledger.transactions.retrieve(req.params.id)
     const connectorEntry = await Connector.charges.retrieveAPI(req.params.id, ledgerEntry.gateway_account_id)
-    const parity = diff(connectorEntry, ledgerEntry)
+
+    const ledgerPreparedForDiff = _.omit(ledgerEntry, [
```

Comment:
Can we rename vars so they are clear?

```suggestion
    const ledgerResponseWithoutLedgerSpecificFields = _.omit(ledgerEntry, [
```

Reply:
Done.